website/docs/r/sentinel_alert_rule_fusion.html.markdown

```diff
@@ -49,12 +39,13 @@ resource "azurerm_sentinel_alert_rule_fusion" "example" {
 
 The following arguments are supported:
 
-* `name` - (Required) The name which should be used for this Sentinel Fusion Alert Rule. Changing this forces a new Sentinel Fusion Alert Rule to be created.
-
 * `log_analytics_workspace_id` - (Required) The ID of the Log Analytics Workspace this Sentinel Fusion Alert Rule belongs to. Changing this forces a new Sentinel Fusion Alert Rule to be created.
 
 * `alert_rule_template_guid` - (Required) The GUID of the alert rule template which is used for this Sentinel Fusion Alert Rule. Changing this forces a new Sentinel Fusion Alert Rule to be created.
 
+* `name` - (Optional) The name which should be used for this Sentinel Fusion Alert Rule. Changing this forces a new Sentinel Fusion Alert Rule to be created.
+**Note:** The `name` is deprecated and will be removed in v5.0 version of the provider. The Fusion Alert Rule is enabled by default with the name `BuiltInFusion`, more details could be found [here](https://learn.microsoft.com/en-us/azure/sentinel/configure-fusion-rules#configure-scheduled-analytics-rules-for-fusion-detections).
```

Comment:
This doesn't follow our [guidelines for breaking changes](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/guide-breaking-changes.md#breaking-schema-changes-and-deprecations).

Please also make sure the upgrade guide is updated with this change.

Reply:
sure, sorry for missed that part.